README.md

```diff
@@ -26,31 +26,40 @@
 
 # FingerprintJS Pro Chrome Extension Example
 
+Fingerprint is a device intelligence platform offering 99.5% accurate visitor identification.
+
 This repository contains an example Chrome extension that uses the FingerprintJS FingerprintJS Pro JavaScript agent.
+
+## Requirements
+
+- Chromium based browser (Chrome, Edge, Opera, etc.)
+
+## Installation
+
 You can install the example chrome extension from [Chrome Web Store](https://chrome.google.com/webstore/detail/fingerprintjs-example-bro/knppbjgkegnlbhddedbilnfmnkdocekn).
 
-### Why it exists
+## Why it exists
 
 The Chrome extension environment has limited capabilities for external scripts running within content scripts. This repository showcases different strategies on how FingeprintJS Pro can be used in this ecosystem. 
 
-### How it works
+## How it works
 
 The solution is split into two parts - **chrome-extension** and **website**.
 
-#### Website
+### Website
 
 The website uses the FingerprintJS Pro JavaScript agent as it normally would, and communicates with the Chrome extension.
 It is hosted on [Github Pages](https://fingerprintjs.github.io/fingerprintjs-pro-chrome-extension-example/).
 
-#### Chrome Extension
+### Chrome Extension
 
 Provides sample extension that consists of popup page, background, and content script.
 
-### Strategies
+## Strategies
```

Comment:
It seems both these strategies are related to the "Chrome Extension" environment, correct? In that case, the heading level should be lower than "Chrome extension". To make the heading hierarchy work, consider

* Making "Website" and "Chrome extension" into H2s
* Removing "Strategies" H2, just continue the Chrome extension section with "This repository contains..." 
* Now you can keep "New window extension strategy" and "Iframe extension strategy" as H3s and it hierarchically makes sense

Reply:
Great suggestion, done!